timescaledb/how-to-guides/hyperfunctions/function-pipelines.md

```diff
@@ -7,7 +7,14 @@ Pandas, and PromQL.
 <highlight type="warning">
 Experimental features could have bugs! They might not be backwards compatible,
 and could be removed in future releases. Use these features at your own risk,
-and do not use any experimental features in production.
+and do not use any experimental features in production. 
+</highlight>
+
```

Comment:
This, btw, is true of any of our functions requiring sorting as well...including at least `time_weight` and `counter_agg`...it seems a bit stark given that...

Reply:
We use this same comment in a lot of places. If you think we're missing it somewhere, we can add it in.